qml/js/functions.js

```diff
@@ -28,6 +28,11 @@ function formatPrice(price, locale) {
     return Number(price).toLocaleString(locale, 'f', precision) ;
 }
 
+function formatCurrencyPrice(price, locale) {
```

Comment:
The usage of the method seems to be missing - at the moment i try to generically determine how many fractions shall be used - if for currency the number is fixed - the logic in the qml needs to be updated - however i do not know by heart if the type is available.